ZukoAzula2016/src/org/team1540/zukoazula/DriveCode.java

```diff
@@ -39,6 +39,7 @@
     private static final Behavior teleop = behaviors.addBehavior("Teleop", FRC.inTeleopMode());
     private static final BooleanCell pitModeEnable = new BooleanCell();
     private static final Behavior pit = behaviors.addBehavior("Pit Mode", pitModeEnable.andNot(FRC.isOnFMS()));
+    private static final Behavior challenge = behaviors.addBehavior("Challenge Brake", ChallengeBrake.driveBrake);
```

Comment:
Clarity to the driver. The behavior arbitrator can publish an RConf view for the PoultryInspector, which would specify which mode is active.